conf/modules_nanopore.config

```diff
@@ -0,0 +1,27 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Config file for defining DSL2 per module options and publishing paths
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Available keys to override module options:
+        ext.args   = Additional arguments appended to command in module.
+        ext.args2  = Second set of arguments appended to command in module (multi-tool modules).
+        ext.args3  = Third set of arguments appended to command in module (multi-tool modules).
+        ext.prefix = File name prefix for output files.
+----------------------------------------------------------------------------------------
+*/
+
+process {
+    withName: FLYE {
+        ext.args = [
+            "--meta",
+            "--scaffold"
+        ].join(' ').trim()
+    }
+
+    withName: MINIMAP2_ALIGN {
+        ext.args = [
+            "-x map-ont",
+            "-a" //output in SAM format
```

Comment:
It has to be SAM - I spent quite a bit of time fiddling with this, racon doesn't support BAM and yes I agree it would save a lot of space. It's been asked pretty frequently on the racon github to support BAM for this reason but I don't think the developer is going to do it. I think the workaround for space would be at the end of the workflow after making sure everything ran successfully convert SAM > BAM so still have that info and get rid of SAM